README.md

```diff
@@ -30,6 +30,12 @@ The complete list of environment variable configuration options is
 2. `BUSTY_COOLDOWN_SECS` - Number of seconds between songs (default = 10)
 3. `BUSTY_ATTACHMENT_DIR` - Directory to save attachments (default = attachments)
 4. `BUSTY_DJ_ROLE` - Name of role with permissions to run commands (default = bangermeister)
+5. `BUSTY_CUSTOM_EMOJI_FILEPATH` - The path to a file containing a custom list of emoji to display.
```

Comment:
I accidentally didn't save these comments in my first review, so I'm writing it again. First off, here we should show a default value (default = emoji_list) to be consistent. Second off, "the path to a file" is not technically accurate, note the default value does not have a file extension. A more accurate description would be something like "The Python module to import containing the emoji list"

```suggestion
5. `BUSTY_CUSTOM_EMOJI_FILEPATH` - The Python module to import containing the emoji list (default = emoji_list)
```
If this change is made then line 38 should also be reworded from `the path of the new file` to something like `the name of the new module`.